cmd/podman/common/completion.go

```diff
@@ -1522,6 +1523,29 @@ func AutocompleteCgroupManager(cmd *cobra.Command, args []string, toComplete str
 	return types, cobra.ShellCompDirectiveNoFileComp
 }
 
+// AutocompleteContainersConfModules- Autocomplete containers.conf modules.
+func AutocompleteContainersConfModules(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
functionality wise we could move this into c/common/pkg/completion as it would be closer to the source

Then if other projects (buildah) ever support modules they could use it as well

but either way not a blocker for me